### DISCUSSION TITLE
How do the PR stats work?

### DISCUSSION THREAD

**Discussion by ionknowmyname** [anuraghazra/github-readme-stats#1686] [Q&A] [answered]
I've made a couple of PR's that have been approved and merged, yet it keeps showing me 0 PR's for my github. And my commits aren't accurate. I've made more commits than what's showing there for this year

**Comment by rickstaa** [comment, marked as answer]
@ionknowmyname Thanks for your interest in our tool. The PRs update according to this graphQL query:

```graphql
query userInfo($login: String!) {
  user(login: $login) {
    repositories(first: 100, orderBy: {direction: DESC, field: STARGAZERS}) {
      totalCount
      nodes {
        name
        pullRequests(first: 1) {
          totalCount
        }
      }
    }
  }
}
```

```graphql
{"login": "ionknowmyname"}
```

You can test it out on https://docs.github.com/en/graphql/overview/explorer. For your account, I see that none of your Public repositories has a pull request. As a result, when using the public Vercel instance, which has no permissions to get data about your private repositories, the PR on the card are shown as `0`. If the pull requests you made are on a private repository please deploy a Vercel instance yourself and give it access to your private repositories using a Personal Access Token (see https://github.com/anuraghazra/github-readme-stats#deploy-on-your-own-vercel-instance).

Also please be aware that we use a cache to update the card so it might take up to 4 hours for your PR to show up (see https://github.com/anuraghazra/github-readme-stats#common-options).

**Reply by ionknowmyname** [reply]
Okay got it, my PR's are on a private repo. Let me go through the link you sent, and set it up for the private repos. Thanks @rickstaa
